pom.xml

```diff
@@ -40,6 +51,11 @@
             <artifactId>plexus-utils</artifactId>
             <version>3.1.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.adoptopenjdk</groupId>
+            <artifactId>jsplitpkgscan</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Would prefer to see this at 1.0.0-SNAPSHOT for future semVer, but we can sort that later.